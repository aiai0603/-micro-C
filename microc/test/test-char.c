void main(int n) {
  
    char a;
    a = '3';
    print("%c",a);

}

