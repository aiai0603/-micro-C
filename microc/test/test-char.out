24 19 1 5 25 15 1 13 0 1 1 27 51 12 15 -1 13 0 1 1 11 23 21 2